src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -36,6 +36,7 @@
 public class EditCommand extends Command {
 
     public static final String COMMAND_WORD = "edit";
+    public static final String ARCHIVE_COMMAND_WORD = "archive";
```

Comment:
I believe you should follow the standard of this project and create a new `ArchiveCommand` class for it.